### PR TITLE
Add Administrator role for creator

### PR DESCRIPTION
**Pull request by RushanNanayakkara**

### CODE REVIEW

File: components/org.wso2.carbon.identity.organization.management.role.management.service/src/main/java/org/wso2/carbon/identity/organization/management/role/management/service/constant/RoleManagementConstants.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ public class RoleManagementConstants {
     public static final String UNION_SEPARATOR = " UNION ALL ";
 
     public static final String ORG_CREATOR_ROLE = "org-creator";
+    public static final String ORG_ADMINISTRATOR_ROLE = "Administrator";
```
Comment (sadilchamishka) [inline components/org.wso2.carbon.identity.organization.management.role.management.service/src/main/java/org/wso2/carbon/identity/organization/management/role/management/service/constant/RoleManagementConstants.java:45]:
```suggestion
    public static final String ORG_ADMINISTRATOR_ROLE = "administrator";
```
As the org-creator is in simple case. @AnuradhaSK wdyt?

**Review by sadilchamishka — DISMISSED** [review]

File: components/org.wso2.carbon.identity.organization.management.role.management.service/src/main/java/org/wso2/carbon/identity/organization/management/role/management/service/constant/RoleManagementConstants.java (hunk 1 above)
Comment (AnuradhaSK) [reply components/org.wso2.carbon.identity.organization.management.role.management.service/src/main/java/org/wso2/carbon/identity/organization/management/role/management/service/constant/RoleManagementConstants.java:45]: Let's go with Administrator, I think we better rename org-creator to Organisation Creator in a separate effort

**Comment by AnuradhaSK** [comment]
Don't merge until migration script is ready

**Review by AnuradhaSK — DISMISSED** [review]

**Comment by RushanNanayakkara** [comment]
Closing due to changes being duplicated in the following PR[1].
[1] https://github.com/wso2-extensions/identity-organization-management/pull/191

**Pull request closed without merging**